pegasus/sites.v3/code.org/public/administrators.haml

```diff
@@ -20,11 +19,18 @@ theme: responsive_full_width
       %img.rounded-corners.box-shadow-primary{src: "/images/admins-page-top.png", alt: "", style: "position: relative; width: 100%;"}
     .clear
 
+-# Show Curriculum Launch banner if dcdo flag is set to true
 - if !!DCDO.get('curriculum-launch-2024', false)
   = view :curriculum_launch_skinny_banner
 - else
   = view :section_divider_line
 
+-# Show Hour of Code banner if hoc_mode is set to "soon-hoc"
+- if DCDO.get('hoc_mode', 'soon-hoc')
+  = view :hoc_skinny_banner
+- else
+  = view :section_divider_line
+
```

Comment:
Does this mean in the case of `curriculum-launch-2024` being false and `hoc_mode` not being `soon-hoc` we'd have two back to back divider lines? Or will we never have those two settings in that configuration?

Reply:
Technically we could get back to back banners, but I don't think these ddco flags will ever be set at the same time. BUT you're right, the divider line will show when only one banner is hidden so I just removed it and added a light gray background to the section below 😅